packages/rs-drive-abci/src/logging.rs

```diff
@@ -686,34 +763,31 @@ pub fn init_for_tests(verbosity: u8) {
     logger_builder
         .add("default", &config)
         .expect("cannot configure default logger");
+
     logger_builder.build().try_install().ok();
 }
 
 /// Verify log directory path and determine absolute path to log file.
```

Comment:
Please consider the following checks:

```suggestion
/// Verify log file path.
///
/// Ensure that the log file path is correct, that is:
/// - it points to a file, not a directory
/// - if the log file exists, it is writable for the current user
/// - parent directory of the file exists and is writable for current user
/// - path is absolute
```

hint: chatgpt seems to be quite good with tasks like this ;)